src/tests/pbc-test.cpp

```diff
@@ -56,18 +56,18 @@ int main(int argc, char** argv)
   for (size_t i = 0; i < systems.size(); i++) {
     // Get NB energy for no_pbc and for pbc
     // First no PBC
-    std::vector<double> xyz = systems[i].GetOriginalOrderSysXyz();
-    std::vector<std::string> ats = systems[i].GetOriginalOrderSysAtNames();
-    double e1b = systems[i].Get1B(false);
-    double e2b = systems[i].Get2B(false);
-    double e3b = systems[i].Get3B(false);
+    std::vector<double> xyz = systems[i].GetRealXyz();
+    std::vector<std::string> ats = systems[i].GetAtomNames();
+    double e1b = systems[i].OneBodyEnergy(false);
+    double e2b = systems[i].TwoBodyEnergy(false);
+    double e3b = systems[i].ThreeBodyEnergy(false);
 
     // Then PBC
     systems[i].SetPBC(true,box);
-    std::vector<double> xyz_pbc = systems[i].GetOriginalOrderSysXyz();
-    double e1b_pbc = systems[i].Get1B(false);
-    double e2b_pbc = systems[i].Get2B(false);
-    double e3b_pbc = systems[i].Get3B(false);
+    std::vector<double> xyz_pbc = systems[i].GetRealXyz();
```

Comment:
+1 for better names. Something to keep in mind is that usually C++ uses CamelCase for classes and snake_case for functions. It isn't hard and set as it is in say python, but just make sure you are consistent throughout the code.

Reply:
Do you mean it would be better to use, let's say, Set_xyz() rather than SetXyz? I think in the code I have been following "more or less" the google c++ standards (see [the naming section](https://google.github.io/styleguide/cppguide.html#Naming) ) 

Do you have any other preference? Or you mention this because it will be easier to link with python?